# Albanian translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1045
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2021-07-26 13:07-0700\n"
"PO-Revision-Date: 2019-06-18 17:54+0000\n"
"Last-Translator: Vilson Gjeci <vilsongjeci@gmail.com>\n"
"Language-Team: Albanian <sq@li.org>\n"
"Language: sq\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:46+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"
"X-Poedit-Bookmarks: 126,-1,-1,-1,-1,-1,-1,-1,-1,-1\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Serveri për %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Serveri Kryesor"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Servera të personalizuar"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fshije paketën në gjendje të keqe"
msgstr[1] "Fshiji paketat në gjendje të keqe"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Paketa '%s' është në një gjendje të paqëndrueshme dhe ka nevojë të "
"riinstalohet, por nuk është gjetur asnjë arkiv për të. Dëshironi ta hiqni "
"këtë paketë tani për të vazhduar?"
msgstr[1] ""
"Paketat '%s' janë në një gjendje të paqëndrueshme dhe kanë nevojë të "
"riinstalohen, por nuk është gjetur asnjë arkiv për to. Dëshironi t'i hiqni "
"këto paketa tani për të vazhduar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "Serveri mund të jetë i mbingarkuar"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Paketat e dëmtuara"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Sistemi juaj përmban paketa defekt, të cilat nuk mund të riparohen me këtë "
"softuer.Ju lutemi riparoni këtë me Synaptic ose apt-get, para se të shkoni "
"përpara."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Paketat e dëmtuara"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Ky ka mundësi të jetë një problem kalimtar, ju lutemi provojeni përsëri më "
"vonë."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "Nuk mundëm të llogarisim përditësimin"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "Gabim gjatë vërtetimit të origjinalitetit të disa paketave."

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Disa paketa nuk mund të vërtetoheshin për origjinalitet.Kjo ka si pasojë "
"problemet me rrjetin.Ju lutemi provoni më vonë edhe një herë.Paketat e "
"mëposhtme nuk mund të vërtetohen për nga origjinaliteti."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Paketa '%s' është shënuar për tu hequr, por është në listën e zezë të heqjes."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Paketa thelbësore '%s' është shënuar për tu hequr."

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Duke u përpjekur të istaloj versionin e listës së zezë '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "'%s' nuk mund të instalohet"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Ishte i pamundur instalimi i një pakete të kërkuar. Ju lutemi ta raportoni "
"këtë si një gabim duke përdorur 'ubuntu-bug ubuntu-release-upgrader-core' në "
"një terminal."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "Can't guess meta-package"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Sistemi juaj nuk përmban një paketë %s ose %s dhe nuk është e mundur të "
"dallojmë kë version të Ubuntu keni.\n"
" Ju lutemi që fillimisht të instaloni një nga paketat e mësipërme duke "
"përdorur synaptic ose apt-get para se të vazhdoni."

#: ../DistUpgrade/DistUpgradeController.py:112
msgid "Reading cache"
msgstr "Duke lexuar depozitën"

#: ../DistUpgrade/DistUpgradeController.py:219
msgid "Unable to get exclusive lock"
msgstr "Është e pamundur të merret një hyrje ekskluzive"

#: ../DistUpgrade/DistUpgradeController.py:220
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Kjo dmth se me siguri, një tjetër Paketmanager është aktiv (si psh apt-get "
"ose aptitude). Ju lutemi mbylleni së pari këtë aplikacion."

#: ../DistUpgrade/DistUpgradeController.py:278
msgid "Upgrading over remote connection not supported"
msgstr "Përditësimi nga një lidhje e largët nuk suportohet"

#: ../DistUpgrade/DistUpgradeController.py:279
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Ju po e nisni përditësimin nga një lidhje e largët ssh me një frontend që "
"nuk e suporton atë. Ju lutemi të provoni një përditësim në mënyrë teksti me "
"'do-release-upgrade'.\n"
"\n"
"Përditësimi do të mbyllet tani. Ju lutemi provojeni pa ssh."

#: ../DistUpgrade/DistUpgradeController.py:293
msgid "Continue running under SSH?"
msgstr "Të vazhdojmë punën nën SSH?"

#: ../DistUpgrade/DistUpgradeController.py:294
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Ky seksion duket se është nisur me ssh. Nuk rekomandohet për momentin të "
"kryeni një përditësim me ssh sepse në rast dështimi është e vështirë të "
"rekuperoni.\n"
"\n"
"Nëse vazhdoni, një ssh daemon shtesë do të nisë në portin '%s'.\n"
"Dëshironi të vazhdoni?"

#: ../DistUpgrade/DistUpgradeController.py:308
msgid "Starting additional sshd"
msgstr "Po nisim sshd shtesë"

#: ../DistUpgrade/DistUpgradeController.py:309
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Për ta bërë riparimin më të lehtë në rastin e ndonjë dështimi, një sshd "
"shtesë do të nisë në portin '%s'. Nëse diçka nuk shkon si duhet me ssh e "
"nisur, ju mund të vazhdoni të lidheni me atë shtesë.\n"

#: ../DistUpgrade/DistUpgradeController.py:317
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Nëse keni një firewall ju duhet që përkohësisht ta hapni këtë port. Duke "
"qenë se kjo është potencialisht e rrezikshme nuk bëhet automatikisht. Ju "
"mund ta hanpi portin me p.sh.:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:395
#: ../DistUpgrade/DistUpgradeController.py:419
msgid "Can not upgrade"
msgstr "Nuk mund të aktualizoj"

#: ../DistUpgrade/DistUpgradeController.py:396
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Një përditësim nga '%s' në '%s' nuk suportohet me këtë mjet."

#: ../DistUpgrade/DistUpgradeController.py:420
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:449
#, python-format
msgid "Can not write to '%s'"
msgstr "Nuk mund të shkruaj tek '%s'"

#: ../DistUpgrade/DistUpgradeController.py:450
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Nuk është e mundur të shkruaj në drejtorinë e sistemit '%s' në sistemin "
"tuaj. Përditësimi nuk mund të vazhdojë.\n"
"Ju lutemi ta bëni të shkruajtshme drejtorinë tuaj të sistemit."

#: ../DistUpgrade/DistUpgradeController.py:617
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "Jo për njerëzit gjatë nivelit të zhvillimit të versionit %s"

#: ../DistUpgrade/DistUpgradeController.py:717
#, python-format
msgid "disabled on upgrade to %s"
msgstr "u çaktivizua në përditësimin e %s"

#: ../DistUpgrade/DistUpgradeController.py:776
msgid "No valid sources.list entry found"
msgstr "Nuk u gjet një hyrje e vlefshme e sources.list"

#: ../DistUpgrade/DistUpgradeController.py:777
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Gjatë skanimit të informacionit të magazinës tuaj nuk gjeta asnjë hyrje për "
"%s.\n"
"\n"
"Një përditësim mund të mos jetë e suksesshëm.\n"
"\n"
"Do të vazhdosh gjithësesi?"

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "No valid mirror found"
msgstr "Nuk u gjet asnjë lidhje e vlefshme"

#: ../DistUpgrade/DistUpgradeController.py:788
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Gjatë skanimit të informacionit të magazinës suaj nuk u gjet asnjë "
"informacion hyrjeje përpërditësimin. Kjo mund të ndodhë nëse përdorni një "
"hapësirë të brendshme ose nëse informacioni i pasqyrës është i pa "
"përditësuar.\n"
"\n"
"Dëshironi ta rishkruani skedarin 'sources.list' gjithësesi? Nëse zgjidhni "
"'Po' këtu ajo do të përditësojë të gjitha '%s' në hyrjet '%s'.\n"
"Nëse zgjidhni 'Jo' përditësimi do anullohet."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:808
msgid "Generate default sources?"
msgstr "Të gjenerojmë resurset e parazgjedhura?"

#: ../DistUpgrade/DistUpgradeController.py:809
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Pasi skanuam 'sources.list' nuk u gjet një hyrje e vlefshme '%s'.\n"
"\n"
"A duhet të shtohen hyrjet e parazgjedhura për '%s'? Nëse zgjidhni 'Jo', "
"përditësimi do të anullohet."

#: ../DistUpgrade/DistUpgradeController.py:844
#: ../DistUpgrade/DistUpgradeController.py:850
msgid "Repository information invalid"
msgstr "Informacioni i magazinës është i pavlefshëm"

#: ../DistUpgrade/DistUpgradeController.py:845
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Përditësimi i informacionit të magazinës krijoi një skedar të apvlefshëm "
"kështu që një proçes i raportimit të gabimit u krijua."

#: ../DistUpgrade/DistUpgradeController.py:851
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Përditësimi i informacionit të magazinës krijoi një skedar të pavlefshëm. "
"Për të raportuar një defekt instalo apport dhe pastaj ekzekuto 'apport-bug "
"ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:860
msgid "Third party sources disabled"
msgstr "Burimet e palëve të treta janë çaktivizuar"

#: ../DistUpgrade/DistUpgradeController.py:861
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Disa hyrje nga palë të treta në sources.list u çaktivizuan. Ju mund t'i "
"riaktivizoni ato pas përditësimit me mjetin 'parametrat e programeve' ose me "
"menaxhuesin tuaj të paketave."

#: ../DistUpgrade/DistUpgradeController.py:903
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paketë në gjendje të paqëndrueshme"
msgstr[1] "Paketa në gjendje të paqëndrueshme"

#: ../DistUpgrade/DistUpgradeController.py:906
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Paketa '%s' është në një gjendje jo të qëndrueshme dhe duhet riinstaluar, "
"por për të nuk mund të gjendet asnjë arkiv. ju lutemi riinstalojeni paketën "
"në mënyrë manuale ose hiqeni nga sistemi."
msgstr[1] ""
"Paketat '%s' janë në një gjendje jo të qëndrueshme dhe duhen riinstaluar, "
"por për to nuk mund të gjendet asnjë arkiv. ju lutemi riinstalojini paketat "
"në mënyrë manuale ose hiqini nga sistemi."

#: ../DistUpgrade/DistUpgradeController.py:969
msgid "Error during update"
msgstr "Gabim gjatë përditësimit"

#: ../DistUpgrade/DistUpgradeController.py:970
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Ndodhi një problem gjatë përditësimit. Ky zakonisht është një lloj problemi "
"me rrjetin, ju lutemi kontrolloni lidhjen tuaj me rrjetin dhe provojeni "
"përsëri."

#: ../DistUpgrade/DistUpgradeController.py:1013
#: ../DistUpgrade/DistUpgradeController.py:1018
msgid "EFI System Partition (ESP) not usable"
msgstr "Ndarja EFI e Sistemit (ESP) nuk është e përdorshme"

#: ../DistUpgrade/DistUpgradeController.py:1014
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"Sistemi juaj i ndarjes EFI (ESP) nuk është montuar në /boot/efi. Ju lutemi "
"të siguroheni që është konfiguruar siç duhet dhe provojeni përsëri."

#: ../DistUpgrade/DistUpgradeController.py:1019
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"Sistemi juaj i ndarjes EFI (ESP) i montuar në /boot/efi nuk mund të "
"shkruhet. Ju lutemi ta montoni këtë particion  për lexim-shkrim dhe "
"provojeni përsëri."

#: ../DistUpgrade/DistUpgradeController.py:1026
msgid "Not enough free disk space"
msgstr "Nuk ka hapësirë të lirë sa duhet"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1029
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"Përditësimi u abortua. Përditësimi ka nevojë për një total prej {str_total} "
"hapësire të lirë në diskun '{str_dir}'. Ju lutemi të çlironi të paktën "
"{str_needed} hapësirë disku në '{str_dir}'. {str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1034
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Hiqi paketat e përkohshme të instalimeve të mëparshme duke përdorur 'sudo "
"apt clean'."

#: ../DistUpgrade/DistUpgradeController.py:1036
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"Ju mund t'i hiqni kernelat e vjetër duke përdorur 'sudo apt autoremove' dhe "
"ju mund të vendosni gjithashtu COMPRESS=xz në /etc/initramfs-tools/initramfs."
"conf për t'a zvogëluar përmasën e initramfs."

#: ../DistUpgrade/DistUpgradeController.py:1041
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Zbrazeni koshin e mbeturinave dhe hiqni paketat e përkohshme të instalimeve "
"të mëparshme duke përdorur 'sudo apt-get clean'."

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid "Reboot to clean up files in /tmp."
msgstr "Rindize për të pastruar failet në /tmp."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1083
#: ../DistUpgrade/DistUpgradeController.py:1925
#: ../DistUpgrade/DistUpgradeController.py:1977
msgid "Calculating the changes"
msgstr "Duke llogaritur ndryshimet"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1127
msgid "Do you want to start the upgrade?"
msgstr "Dëshironi të nisni përditësimin?"

#: ../DistUpgrade/DistUpgradeController.py:1146
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1218
msgid "Upgrade canceled"
msgstr "Përditësimi u anullua"

#: ../DistUpgrade/DistUpgradeController.py:1219
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Përditësimi do të anullohet tani dhe do të kthehet gjendja origjinale e "
"sistemit. Ju mund ta rinisni përditësimin më vonë."

#: ../DistUpgrade/DistUpgradeController.py:1225
#: ../DistUpgrade/DistUpgradeController.py:1370
msgid "Could not download the upgrades"
msgstr "Nuk mund të aktualizohen, shkarkimet"

#: ../DistUpgrade/DistUpgradeController.py:1226
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Përditësimi u anullua. Ju lutemi të kontrolloni lidhjen tuaj me internatin "
"apo median tuaj të instalimit dhe ta provoni përsëri. Të gjithë skedarët e "
"shkarkuar deri në këtë moment janë mbajtur."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1319
#: ../DistUpgrade/DistUpgradeController.py:1358
#: ../DistUpgrade/DistUpgradeController.py:1474
msgid "Error during commit"
msgstr "Gabim gjatë vendosjes"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1321
#: ../DistUpgrade/DistUpgradeController.py:1360
#: ../DistUpgrade/DistUpgradeController.py:1519
msgid "Restoring original system state"
msgstr "Duke kthyer gjendjen origjinale të sistemit"

#: ../DistUpgrade/DistUpgradeController.py:1322
#: ../DistUpgrade/DistUpgradeController.py:1337
#: ../DistUpgrade/DistUpgradeController.py:1361
msgid "Could not install the upgrades"
msgstr "Nuk mund të instalohen, aktualizimet"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1327
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Përditësimi u anullua. Sistemi juaj mund të jetë në një gjendje të "
"paqëndrueshme. Tani do të nisë rekuperimi (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1332
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Ju lutemi ta raportoni këtë gabim në një shfletues interneti tek http://bugs."
"launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug dhe "
"bashkangjitni skedarët tek /var/log/dist-upgrade/ në raportin e gabimit.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1371
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Përditësimi u anullua. Ju lutemi të kontrolloni lidhjen tuaj me internetin "
"apo median e instalimit dhe të provoni përsëri. "

#: ../DistUpgrade/DistUpgradeController.py:1399
msgid "Searching for obsolete software"
msgstr "Duke kërkuar për programe të vjetra"

#: ../DistUpgrade/DistUpgradeController.py:1462
msgid "Remove obsolete packages?"
msgstr "Ti heqim paketat e vjetra?"

#: ../DistUpgrade/DistUpgradeController.py:1463
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Mbaj"

#: ../DistUpgrade/DistUpgradeController.py:1463
msgid "_Remove"
msgstr "_Hiqe"

#: ../DistUpgrade/DistUpgradeController.py:1475
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Një problem ndodhi gjatë pastrimit. ju lutemi shikoni mesazhin e mësipërm "
"për më tepër informacion. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1557
msgid "Required depends is not installed"
msgstr "Vartësitë e kërkuara nuk janë instaluar"

#: ../DistUpgrade/DistUpgradeController.py:1558
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Vartësia e kërkuar '%s' nuk është instaluar. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1791
#: ../DistUpgrade/DistUpgradeController.py:1875
msgid "Checking package manager"
msgstr "Duke kontrolluar menaxhuesin e paketave"

#: ../DistUpgrade/DistUpgradeController.py:1797
#: ../DistUpgrade/DistUpgradeController.py:1803
msgid "Preparing the upgrade failed"
msgstr "Përgatitja e përditësimit dështoi"

#: ../DistUpgrade/DistUpgradeController.py:1798
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Përgatitja e sistemit për përditësim dështoi ndaj një proçes i raportimit të "
"gabimit po niset."

#: ../DistUpgrade/DistUpgradeController.py:1804
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Përgatitja e sistemit për përditësim dështoi. Për të raportuar një defekt "
"instalo apport dhe pastaj ekzekuto 'apport-bug ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:1821
#: ../DistUpgrade/DistUpgradeController.py:1831
msgid "Getting upgrade prerequisites failed"
msgstr "Marrja e kërkesave të përditësimit dështoi"

#: ../DistUpgrade/DistUpgradeController.py:1822
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Sistemi nuk ishte në gjendje të merrte kërkesat paraprake përpërditësim. "
"Përditësimi tani do të abortohet dhe do të kthejë gjendje origjinale të "
"sistemit.\n"
"\n"
"Gjithashtu, një proçes i raportimit të gabimit do të niset."

#: ../DistUpgrade/DistUpgradeController.py:1832
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"Sistemi nuk është në gjendje të gjejë parametrat e duhur për tu pëditësuar. "
"Përditësimi do të anullohet dhe do të rikthehet gjendja origjinale e "
"sistemit.\n"
"\n"
"Për të raportuar një mosfunksionim instalo apport dhe pastaj ekzekuto "
"'apport-bug ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:1866
msgid "Updating repository information"
msgstr "Duke përditësuar informacionin e magazinës"

#: ../DistUpgrade/DistUpgradeController.py:1902
msgid "Invalid package information"
msgstr "Informacion i pavlefshëm i paketave"

#: ../DistUpgrade/DistUpgradeController.py:1903
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1932
#: ../DistUpgrade/DistUpgradeController.py:2038
msgid "Fetching"
msgstr "Duke mbledhur"

#: ../DistUpgrade/DistUpgradeController.py:1939
#: ../DistUpgrade/DistUpgradeController.py:1962
#: ../DistUpgrade/DistUpgradeController.py:1992
#: ../DistUpgrade/DistUpgradeController.py:2042
msgid "Upgrading"
msgstr "Aktualizo"

#: ../DistUpgrade/DistUpgradeController.py:1941
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1942
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1965
#: ../DistUpgrade/DistUpgradeController.py:1981
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1966
#: ../DistUpgrade/DistUpgradeController.py:1982
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2044
#: ../DistUpgrade/DistUpgradeController.py:2050
#: ../DistUpgrade/DistUpgradeController.py:2061
msgid "Upgrade complete"
msgstr "Aktualizimi përfundoi"

#: ../DistUpgrade/DistUpgradeController.py:1998
#: ../DistUpgrade/DistUpgradeController.py:2045
#: ../DistUpgrade/DistUpgradeController.py:2051
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Përditësimi u kompletua por pati gabime gjatë proçesit të përditësimit."

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "System upgrade is complete."
msgstr "Aktualizimi i sistemit është komplet"

#: ../DistUpgrade/DistUpgradeController.py:2062
msgid "The partial upgrade was completed."
msgstr "Përditësimi i pjesshëm nuk u kompletua."

#: ../DistUpgrade/DistUpgradeController.py:2091
msgid "Lock screen disabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2092
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Nuk mund të gjejmë shënimet e versionit"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Serveri mund të jetë i mbingarkuar. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Nuk mund të shkarkojmë shënimet e versionit"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherCore.py:75
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Ju lutemi kontrolloni lidhjen tuaj me internetin."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:70
#, fuzzy
msgid "Could not find the release announcement"
msgstr "Nuk mund të gjejmë shënimet e versionit"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:71
#, fuzzy
msgid "The server may be overloaded."
msgstr "Serveri mund të jetë i mbingarkuar. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:74
#, fuzzy
msgid "Could not download the release announcement"
msgstr "Nuk mund të shkarkojmë shënimet e versionit"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:80
#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Vazhdo [pJ] "

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeFetcherCore.py:82
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "p"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:99
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "identifiko '%(file)s' kundrejt '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:125
#, python-format
msgid "extracting '%s'"
msgstr "duke ekstraktuar '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:146
#: ../DistUpgrade/DistUpgradeFetcherCore.py:147
msgid "Could not run the upgrade tool"
msgstr "Nuk mund ta nisim mjetin e përditësimit"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:148
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Ky ka mundësi të jetë një gabim në mjetin e përditësimit. Ju lutemi ta "
"raportoni si një gabim duke përdorur komandën 'ubuntu-bug ubuntu-release-"
"upgrader-core'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:225
msgid "Upgrade tool signature"
msgstr "Përditësoni firmën e mjeteve"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:232
msgid "Upgrade tool"
msgstr "Mjeti përditësues"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
msgid "Failed to fetch"
msgstr "Dështuam në marrje"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:267
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Marrja e përditësimit dështoi. Mund të jetë një problem rrjeti. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid "Authentication failed"
msgstr "Identifikimi dështoi"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:272
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Identifikimi i përditësimit dështoi. Mund të ketë një problem me rrjetin ose "
"me serverin. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
msgid "Failed to extract"
msgstr "Dështuam në nxjerrje"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Nxjerrja e përditësimit dështoi. Mund të ketë një problem me rrjetin apo me "
"serverin. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid "Verification failed"
msgstr "Verifikimi dështoi"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:284
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Verifikimi i përditësimit dështoi. Mund të ketë një problem me rrjetin apo "
"me serverin. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:298
#: ../DistUpgrade/DistUpgradeFetcherCore.py:304
msgid "Can not run the upgrade"
msgstr "Nuk mund të nisim përditësimin"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:299
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Kjo zakonisht shkaktohet nga një sistem ku /tmp montohet noexec. Ju lutemi "
"ta rimontoni pa noexec dhe nisni përditësimin përsëri."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:305
#, python-format
msgid "The error message is '%s'."
msgstr "Mesazhi i gabimit është '%s'."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Përditëso"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Anulo"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Shënime Mbi Versionin"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Aktualizo"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Duke shkarkuar skedarë shtesë të paketave..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Skedari %s nga %s në %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "skedari %s nga %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Ju lutem Fut '%s' Në ngases '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Ndryshim i Medias"

#: ../DistUpgrade/DistUpgradeQuirks.py:230
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Hardueri juaj i garfikëve mund të mos suportohet plotësisht në Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:232
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"Nisja e ambientit të desktopit 'unity' nuk suportohet plotësisht nga "
"hardware juaj i grafikëve. Ndoshta mund të përfundoni në një ambient shumë "
"të ngadaltë pas përditësimit. Këshilla jonë është të mbani versionin LTS për "
"momentin. Për më tepër informacion shikoni https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D. Prapëseprapë a dëshironi të vazhdoni me "
"përditësimin?"

#: ../DistUpgrade/DistUpgradeQuirks.py:256
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Grafikët tuaj mund të mos suportohen plotësisht në Ubuntu 12.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:258
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Mbështetja në Ubuntu 12.04 LTS pëe hardware tuaj të grafikëve Intel është e "
"kufizuar dhe ju mund të hansi në probleme pas përditësimit. Për më tepër "
"informacion shikoni https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForI8xx Dëshironi të vazhdoni me përditësimin?"

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Përditësimi mund të pakësojë efektet e desktopit, performancën e lojërave "
"dhe të programeve të tjerë me grafikë intensivë."

#: ../DistUpgrade/DistUpgradeQuirks.py:282
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Ky kompjuter për momentin po përdorin driverin e grafikëve AMD 'fglrx'. Nuk "
"ka një version të disponueshëm të këtij driveri që të punojë me kartën tuaj "
"video në Ubuntu 10.04 LTS.\n"
"\n"
"Dëshironi të vazhdoni?"

#: ../DistUpgrade/DistUpgradeQuirks.py:321
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:322
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:337
msgid "No ARMv6 CPU"
msgstr "Jo ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:338
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Sistemi juaj përdor një ARM CPU i cili është më i vjetër se arkitektura "
"ARMv6. Të gjitha paketat në KArmic janë ndërtuar me përmirësime që kërkojnë "
"ARMv6 si arkitekturë minimale. Nuk është e mundur ta përditësoni sistemin "
"tuaj në një version të ri të Ubuntu me këtë hardware."

#: ../DistUpgrade/DistUpgradeQuirks.py:358
msgid ""
"Sorry, this storage driver is not supported in kernels for newer releases"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:360
#, python-format
msgid ""
"There will not be any further Ubuntu releases that provide kernel support "
"for the aufs storage driver.\n"
"\n"
"Please ensure that none of your containers are using the aufs storage "
"driver, remove the directory %s and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:380
msgid "No init available"
msgstr "Nuk ka init të disponueshëm"

#: ../DistUpgrade/DistUpgradeQuirks.py:381
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Sistemi juaj duket se është një ambient i virtulizuar pa një init daemon, "
"pra Linux-VServer. Ubuntu 10.04 LTS nuk mund të funksionojë brenda këtij "
"lloj ambienti, sepse kërkon që fillimisht ju t'i bëni një përditësim "
"konfigurimit të makinës suaj virtuale.\n"
"\n"
"Jeni i sigurtë që dëshironi të vazhdoni?"

#: ../DistUpgrade/DistUpgradeQuirks.py:422
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:423
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:554
msgid "Connection to the Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:555
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:570
msgid "Connection to Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:571
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:580
msgid "Outdated snapd package"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:604
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:635
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:641
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:645
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:648
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:677
msgid "PAE not enabled"
msgstr "PAE nuk është aktivizuar"

#: ../DistUpgrade/DistUpgradeQuirks.py:678
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"Sistemi yt përdor një CPU që nuk e ka PAE të aktivizuar. Ubuntu i suporton "
"sistemet jo-PAE deri tek Ubuntu 12.04. Për të kaluar në një version të "
"mëvonshëm të Ubuntu, ju duhet të aktivizoni PAE (nëse është e mundur) "
"shikoni:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1050
msgid "Checking for installed snaps"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Përdor frontend. E disponueshme për momentin: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "*DEPRECATED* this option will be ignored"
msgstr "*DEPRECATED* ky opsion do të shpërfillet."

#: ../DistUpgrade/DistUpgradeMain.py:68
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Kryej një përditësim të pjesshëm (pa rishkruar sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:71
msgid "Disable GNU screen support"
msgstr "Çaktivizo mbështetjen për ekranin e GNU"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Set datadir"
msgstr "Vendos datadir"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "Upgrade to the development release"
msgstr "Përditësoni drejt versionit në zhvillim"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Marrja u kompletua"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Duke marrë skedarin %li e %li në %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Rreth %s të mbetura"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Duke marrë skedarin %li të %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Duke aplikuar ndryshimet"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "probleme vartësie - po e lëmë të pakonfiguruar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Nuk mund të instalojmë '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Përditësimi do të vazhdojë por paketa '%s' mund të mos jetë në gjendje pune. "
"ju lutemi të konsideroni lëshimin e një raporti të defektit rreth saj."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Zëvendëso skedarin e përshtatur të konfigurimit\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Ju do të humbisni çdo ndryshim që i keni bërë këtij skedari konfigurimi nëse "
"vendosni që ta zëvendësoni atë me një version më të ri."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "Komanda 'diff' nuk u gjet"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Ndodhi një gabim fatal"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Ju lutemi ta raportoni këtë gabim (nëse nuk e keni bërë tashmë) dhe "
"përfshini skedarët /var/log/dist-upgrade/main.log dhe /var/log/dist-upgrade/"
"apt.log në raportin tuaj. Përditësimi u anullua.\n"
"Origjinali i sources.list u ruajt tek /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl-c i shtypur"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Kjo mund të ndërpresë këtë operacion dhe bërë që sistemi të jetë jo i "
"qëndrueshëm. A jeni të sigurtë, se dëshironi ta bëni këtë?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Për të parandaluar humbjen e të dhënave mbyllni të gjitha programet dhe "
"dokumentat e hapura."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Nuk suportohet më nga Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Ulje në Shkallë (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Hiqe (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Nuk nevojitet më (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Instalo (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Përditëso (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Shfaq Ndryshimin >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Fshihe Ndryshimin"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Gabim"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Mbylle"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Trego Terminalin >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Fshihe Terminalin"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Informacion"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Nis Përditësimin"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detajet"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Nuk suportohet më %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Largo %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Hiqe (ishte vetë instaluar) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Instalo %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Aktualizo %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Kërkohet rinisja"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Rinisni sistemin për të kompletuar përditësimin</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Rinis tani"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Ta anullojmë përditësimin që po kryhet?</big></b>\n"
"\n"
"Sistemi mund të ngelet në një gjendje të paqëndrueshme nëse e anulloni "
"përditësimin. Ju këshillojmë që ta vazhdoni përditësimin."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Anullo Aktualizimin?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li ditë"
msgstr[1] "%li ditët"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li orë"
msgstr[1] "%li orë"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minutë"
msgstr[1] "%li minuta"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekondë"
msgstr[1] "%li sekonda"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Ky shkarkim do të zgjasë rreth %s me një lidhje 1Mbit DSL dhe rreth %s me "
"një modem 56k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Ky shkarkim do të dojë %s me lidhjen tuaj. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Duke u përgatitur për përditësim"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Duke marrë kanalet e reja të programeve"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Duke marrë paketat e reja"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Duke instaluar përditësimet"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Duke pastruar"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paketa e instaluar nuk suportohet më nga Canonical. Ju mund të "
"merrni prapëseprapë mbështetje nga bashkësia."
msgstr[1] ""
"%(amount)d paketat e instaluara nuk suportohen më nga Canonical. Ju mund të "
"merrni prapëseprapë mbështetje nga bashkësia."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d paketë do të hiqet."
msgstr[1] "%d paketa do të hiqen."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d paketë e re do të instalohet."
msgstr[1] "%d paketa të reja do të instalohen."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d paketë do të përditësohet."
msgstr[1] "%d paketa do të përditësohen."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Juve ju duhet të shkarkoni një total prej %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Instalimi i përditësimit mund të zgjasë disa orë. Pasi shkarkimi të ketë "
"përfunduar, proçesi nuk mund të anullohet."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Marrja dhe instalimi i përditësimit mund të zgjasë disa orë. Pasi shkarkimi "
"të ketë përfunduar, proçesi nuk mund të anullohet."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "Heqja e paketave mund të zgjasë disa orë. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "Programet në këtë kompjuter janë të përditësuara."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Ato aktualizime nuk jan te gatshem per sistemin tuaj. Aktualizimet nuk mun "
"te largohen."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Kërkohet rinisja"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Përditësimi përfundoi dhe kërkohet një rindezje. Dëshironi ta bëni këtë tani?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progresi: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Ju lutemi ta raportoni këtë gabim dhe përfshini skedarët /var/log/dist-"
"upgrade/main.log dhe /var/log/dist-upgrade/apt.log në raportin tuaj. "
"Përditësimi u anullua.\n"
"Origjinali i sources.list u ruajt tek /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Duke Ndërprerë"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Demoted:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Për të vazhduar ju lutemi të shtypni [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detajet [d]"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "j"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Nuk suportohet më: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Fshij: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Instalo: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Aktualizo: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Vazhdo [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Për të mbaruar aktualizimin, kërkohet një rinisje.\n"
"Në qoftëse ju zgjithni 'y' sistemi do të rinis."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Duke shkarkuar skedarin %(current)li nga %(total)li me %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Duke shkarkuar skedarin %(current)li nga %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Shfaq progresin e skedarëve individualë"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Ndërprej aktualizimin"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Duke Rinisur Përditësimin"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Ta fshijmë përditësimin e nisur?</big></b>\n"
"\n"
"Sistemi mund të gjendet në një gjendje të paqëndrushme nëse e anulloni "
"përditësimin. ju këshillojmë nxehtësisht që ta rinisni përditësimin."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Starto Aktualizimin"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Zevëndëso"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Dallimet në mes të të dhënave"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Raporto Bug"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Vazhdo"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Ta nisim përditësimin?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Rindizni sistemin për të kompletuar përditësimin</big></b>\n"
"\n"
"Ju lutemi të ruani punën tuaj para se të vezhdoni."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Përditësim i Distribucionit"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Po përditësoj Ubuntu në versionin %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Duke vendosur kanale të reja të programeve"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Duke rinisur kompjuterin"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminali"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Përditëso"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Është i disponueshëm një version i ri i ubuntu. Do të donit ta "
"përditësonit?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Mos e Përditëso"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Më Pyet Më Vonë"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Po, Përditësoje Tani"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Ju e keni mohuar përditësimin për Ubuntu-n e ri"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Ju mund të përditësoni më vonë duke hapur Përditësuesin e Programeve dhe "
"duke klikuar mbi \"Përditëso\"."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Bëni një përditësim të versionit."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Për të përditësuar Ubuntu, ju duhet të identifikoheni."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Kryej një përditësim të pjesshëm"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Për të kryer një përditësim të pjesshëm, ju duhet të identifikoheni."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Shfaq versionin dhe dil"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Direktori që përmban skedarët e të dhënave"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Nis hapin e specifikuar"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Duke nisur një përditësim të pjesshëm"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Duke shkarkuar mjetin për përditësimin e versionit"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Nëse po përdorni versionin e fundit të suportuar, përditësoni në versionin e "
"zhvillimit"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:188
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Përpiqu të përditësosh në versionin e fundit duke përdorur përditësuesin nga "
"$distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Nise në një mënyrë speciale përditësimi.\n"
"Për momentin suportohet 'desktop' për përditësimet e rregullta dhe 'server' "
"për sistemet server."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Kontrollo vetëm nëse një version i ri i distribucionit është i disponueshëm "
"dhe raporto rezultatin me anë të kodit në dalje"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "Opsionet --devel-release dhe --proposed janë"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr ""
"përjashtuese për njera-tjetrën. Ju lutemi të përdorni vetëm një prej tyre."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "Duke kontrolluar për një version të ri të Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Versioni juaj i Ubuntu nuk suportohet më."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Për informacionin e përditësimeve, ju lutemi të vizitoni:\n"
"%(url)s\n"

#: ../do-release-upgrade:155
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:156
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:158
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:159
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:160
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:162
msgid "Upgrades to the development release are only "
msgstr "Përditësimet për versionin e zhvillimit janë vetëm "

#: ../do-release-upgrade:163
msgid "available from the latest supported release."
msgstr "të disponueshme nga versioni i fundit i suportuar."

#: ../do-release-upgrade:165
msgid "No new release found."
msgstr "Nuk u gjet një version i ri."

#: ../do-release-upgrade:170
msgid "Release upgrade not possible right now"
msgstr "Përditësimi i versionit nuk është i mundur për momentin."

#: ../do-release-upgrade:171
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Përditësimi i versionit nuk mund të bëhet siç duhet. Ju lutemi ta provoni "
"përsëri më vonë. Serveri raportoi: '%s'"

#: ../do-release-upgrade:177
#, c-format
msgid "New release '%s' available."
msgstr "Versioni i ri '%s' i disponueshëm."

#: ../do-release-upgrade:178
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Nis 'do-release-upgrade' për të përditësuar atë."

#: ../do-release-upgrade:195
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:211
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:88
msgid "Software Updater"
msgstr "Përditësuesi i Programeve"

#: ../check_new_release_gtk.py:120
#, fuzzy, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(versioni)s Përditësim i Disponueshëm"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:151
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Ju e keni mohuar përditësimin tek ubuntu %s"

#: ../check_new_release_gtk.py:184
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Kontrollo nëse përditësimi në versionin e fundit dlevel është i disponueshëm"

#: ../check_new_release_gtk.py:193
msgid "Add debug output"
msgstr "Shto daljen e kontrollit të gabimeve"

#, fuzzy
#~ msgid "An upgrade is not possible at this time"
#~ msgstr "Përditësimi i versionit nuk është i mundur për momentin."

#~ msgid "Could not calculate sources.list entry"
#~ msgstr "Nuk mundëm të llogarisim hyrjen sources.list"

#~ msgid ""
#~ "Unable to locate any package files, perhaps this is not a Ubuntu Disc or "
#~ "the wrong architecture?"
#~ msgstr ""
#~ "I pamundur lokalizimi i skedarëve të paketave, ndoshta ky nuk është një "
#~ "disk i Ubuntu ose është në arkitekturë e gabuar?"

#~ msgid "Failed to add the CD"
#~ msgstr "Futja e CD dështoi"

#~ msgid ""
#~ "There was a error adding the CD, the upgrade will abort. Please report "
#~ "this as a bug if this is a valid Ubuntu CD.\n"
#~ "\n"
#~ "The error message was:\n"
#~ "'%s'"
#~ msgstr ""
#~ "Pati një gabim në shtimin e CD, përditësimi do të abortohet. Ju lutemi "
#~ "raportojeni këtë si një defekt nëse ky është një CD i vlefshëm i Ubuntu.\n"
#~ "\n"
#~ "Mesazhi i gabimit ishte:\n"
#~ "'%s'"

#~ msgid "Include latest updates from the Internet?"
#~ msgstr "Të përfshijmë edhe përditësimet e fundit nga Interneti?"

#~ msgid ""
#~ "The upgrade system can use the internet to automatically download the "
#~ "latest updates and install them during the upgrade.  If you have a "
#~ "network connection this is highly recommended.\n"
#~ "\n"
#~ "The upgrade will take longer, but when it is complete, your system will "
#~ "be fully up to date.  You can choose not to do this, but you should "
#~ "install the latest updates soon after upgrading.\n"
#~ "If you answer 'no' here, the network is not used at all."
#~ msgstr ""
#~ "Sistemi i përditësimit mund të përdorë internetin për të shkarkuar "
#~ "automatikisht përditësimet e fundit dhe për t'i instaluar ato gjatë "
#~ "përditësimit. Nëse keni një lidhje me rrjetin j'ua rekomandojmë "
#~ "nxehtësisht këtë.\n"
#~ "\n"
#~ "Përditësimi do të zgjasë më shumë, por kur të kompletohet, sistemi juaj "
#~ "do të jetë tërësisht i përditësuar. Ju mund të zgjidhni të mos e bëni "
#~ "këtë, por ju duhet të instaloni përditësimet e fundit me njëherë pas "
#~ "instalimit.\n"
#~ "Nëse përgjigjeni 'jo' këtu, rrjeti nuk do të përdoret fare."

#~ msgid "Failed to add the cdrom"
#~ msgstr "Dështova në shtimin e cdrom"

#~ msgid "Sorry, adding the cdrom was not successful."
#~ msgstr "Më vjen keq, shtimi i cdrom nuk qe i suksesshëm."

#~ msgid "Use the given path to search for a cdrom with upgradable packages"
#~ msgstr ""
#~ "Përdor shkurtoren e dhënë për të kërkuar për një cdrom me paketa të "
#~ "përditësueshme"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Ndodhi një problem i pazgjidhshëm ndërkohë që po llogarisnim "
#~ "përditësimin:\n"
#~ "%s\n"
#~ "\n"
#~ " Kjo mund të shkaktohet nga:\n"
#~ " * Përditësimi në një version testimi të Ubuntu\n"
#~ " * Nisja e versionit të testimit të Ubuntu\n"
#~ " * Paketa jo zyrtare të programeve që nuk jepen nga Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Sistemi juaj nuk prëmban një paketë ubuntu-desktop, kubuntu-desktop, "
#~ "xubuntu-desktop ose edubuntu-desktop dhe nuk kemi mundësi të dallojmë "
#~ "cilin version të Ubuntu dispononi.\n"
#~ " Ju lutemi instaloni fillimisht një nga paketat e mësipërme duke përdorur "
#~ "synaptic apo apt-get para proçedimit."

#~ msgid "Sandbox setup failed"
#~ msgstr "Instalimi i Sandbox dështoi"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Nuk qe e mundur të krijohej ambienti i sandbox."

#~ msgid "Sandbox mode"
#~ msgstr "Mënyra Sandbox"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Ky përditësim është nisur në mënyrën e kutisë së rërës (testimi). Të "
#~ "gjitha ndryshimet janë shkruar tek '%s' dhe do të humbasin pas "
#~ "rindezjes.\n"
#~ "\n"
#~ "*Asnjë* ndryshim i shkruajtur në një drejtori të sistemit nga tani e deri "
#~ "në rindezje nuk do të jetë i përhershëm."

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "Instalimi juaj i python është i dëmtuar. Ju lutemi rregulloni '/usr/bin/"
#~ "python' symlink."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "Paketa 'debsig-verify' është instaluar"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "Përditësimi nuk mund të vazhdojë me atë paketë të instaluar.\n"
#~ "Ju lutemi hiqeni paraprakisht me synaptic ose 'apt-get remove debsig-"
#~ "verify' dhe nisni përditësimin përsëri."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Ndërkohë që po skanonim informacionin e magazinave tuaja nuk u gjet një "
#~ "pasqyrë për përditësim. Kjo mund të ndodhë kur ju keni një pasqyrë të "
#~ "brendshme ose kur informacioni i pasqyrës nuk është i përditësuar.\n"
#~ "Dëshironi ta rishkruani gjithësesi skedarin tuaj 'sources.list'? Nëse "
#~ "zgjidhni 'Po' këtu ai do t'i përditësojë të gjitha hyrjet '%s' tek '%s'.\n"
#~ "Nëse zgjidhni 'Jo' përditësimi do të anullohet."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Ky përditësim u anullua. Përditësimi ka nevojë për një total prej %s "
#~ "hapësirë të lirë në diskun '%s'. Ju lutemi të lironi të paktën %s "
#~ "hapësirë shtesë në diskun '%s'. Zbrazni koshin e mbeturinave dhe hiqni "
#~ "paketat e përkohshme të instalimeve të mëparshme duke përdorur 'sudo apt-"
#~ "get clean'."

#~ msgid "evms in use"
#~ msgstr "evms është në përdorim"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Sistemi juaj përdor menaxhuesin e volumit 'evms' në /proc/mounts. "
#~ "Programi 'evms' nuk suportohet më, ju lutemi fikeni atë dhe nisni "
#~ "përditësimin përsëri kur të keni mbaruar."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Ky kompjuter për momentin po përdorin driverin e grafikëve NVIDIA "
#~ "'nvidia'. Nuk ka një version të disponueshëm të këtij driveri që të "
#~ "punojë me kartën tuaj video në Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Dëshironi të vazhdoni?"

#~ msgid "No i686 CPU"
#~ msgstr "Jo i686 CPU"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "Sistemi juaj përdor një CPU i586 ose një CPU që nuk e ka zgjatimin "
#~ "'cmov'. Të gjitha paketat janë ndërtuar me përmirësime që kërkojnë i686 "
#~ "si arkitekturë minimale. Nuk është e mundur ta përditësoni sistemin tuaj "
#~ "në një version të ri të Ubuntu me këtë hardware."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Po përditësojmë Sandbox duke përdorur aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Testoje përditësimin me mbivedosjen e sandbox aufs"

#~ msgid "No new release found"
#~ msgstr "Nuk u gjet asnjë version i ri"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Për të kryer një përditësim të pjesshëm kërkohet identifikimi"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "Nëse asnjë nga këto nuk aplikohet, atëherë ju lutemi ta raportoni këtë "
#~ "gabim duke përdorur komandën 'ubuntu-bug ubuntu-release-upgrader-core' në "
#~ "një terminal."

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Kërkohet identifikimi për të bërë një përditësim të versionit"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Ndodhi një problem i pazgjidhshëm kur po llogaritej përditësimi.\n"
#~ "\n"
#~ " Kjo mund të shkaktohet nga:\n"
#~ " * Përditësimi në një version të Ubuntu që nuk është akoma gati\n"
#~ " * Përdorimi i një versioni të Ubuntu që nuk është akoma gati\n"
#~ " * Paketa programesh jo zyrtare që nuk jepen nga Ubuntu\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Po përditësoj Ubuntu në versionin 15.04</big></b>"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
#~ msgstr "<b><big>Po përditësoj Ubuntu në versionin 15.10</big></b>"
